packages/jest-snapshot/src/__tests__/throw_matcher.test.ts

```diff
@@ -5,11 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
+
 'use strict';
```

Comment:
ditch this after making the below `require` an `import`

Reply:
You mean 'use strict' ?

Reply:
yes - with ESM strict mode is the default

Reply:
BTW, I think there is no option to destruct with `import = require()`

`import {toThrowErrorMatchingSnapshot} = require('../');`

This gives error